oereb_einzelschutz_denkmal/insert_oereb_einzelschutz_denkmal_tables.sql

```diff
@@ -0,0 +1,570 @@
+        eigentumsbeschraenkung.weiteresthema,
+        transferstruktur_darstellungsdienst.t_id
+    FROM
+    (
+        SELECT
+            DISTINCT ON (artcode, artcodeliste, geometrietyp)
+            eigentumsbeschraenkung.t_basket,
+            eigentumsbeschraenkung.t_datasetname,
+            0::int AS t_seq,        
+            decode('iVBORw0KGgoAAAANSUhEUgAAAEYAAAAjCAYAAAApF3xtAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAD8AAAA/AB2OVKxAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAOoSURBVGiB7ZpfaM1hGMc/PzvGjFLDzGwixVD+xOnsQmOrlcTGldquFq2RzJ+yG5KUpYw0t65WxtXcUGwXXJhIys1xQZSNkWSOzUp6XDzO3vM6e4/fjB3yfuvtfd7v+zzv+zvfnvfP+Z0TCAgeaZiS7Qf4W+GFcSBitYqKIBbL0qNkGffuwevXpi0go6WmRv5b1NRIqhZ+KTnghXHAC+OAF8YBL4wDXhgH3MK8fAnr19tlwwbYtg3a2+HzZ+P76JHxOXvWHqe2VvnNmw0XiylXWQmJhO1//rwZ6+pV5To6DHfxou3/4QNs2qR9Gzcqd+uW8T9xwvY/dsz0dXe7lXHeY+Jx61xPK2VlIm/fqm9Pj+H37bPvB4sWKZ+fb7ggMP4tLYZ//lwkL8/0tbUp39pquPx8kb4+E9PcbPqmTVNuaEikoEC5mTNFEgnlBwc1HkTmzhUZHp7gPWbFCmhthePHYdky5eJxaGkJFZ4R587Bs2dqHzliZ+JYGBoy88bj6RkEMGMGNDaq/ekTXLmidmenxgM0NUFennueUBmzc6fhX70SiUSULymZeMaASG2tPUamjAGN7+0Vqa62+WTGiIj094vk5ipfXq5cNKrt6dNF3ryxn3PCN9+iIi0A79+PO9xCEGjd1QV1dTaXKUYEduyAmzfdMQsWwK5davf2wuXLcP++tuvrYd68jNP82qk05TcdZsuX6wYMMDCg4yaXgAtNTSrEwIC2q6thyZKxfQ8eNPbu3VoHgc07kP3juq0NcnLUbmiAdesy+0ejJrsikfRTMBVr1hjhh4e13rJF98yfIPvCrF4Ne/dCaSmcOhUu5vRpKC6Gw4dh1arMvj9mx6FDoaaI/NxlEnDhgpawWLgQ+vrC+W7dCrNm6X1pzhyoqgoV9msZkzzykntNbq7p+/LF9k22U30mE0GgSw5g6tTQYeMX5to1ePdO7dJSrRcvNifDnTswMqL248fmrdjSpeOeKpsIt5R6evQKnUjA06eGT+70xcW6qV2/Dk+eQFkZrFwJt2/r0QqwZ89vfvQ/i3DCDA7Cw4emnZOjG9+BA4a7dEm/Rz14AC9eaAHNpP37jYj/CNzCzJ+vXwNSEQTKV1VplqSisBDu3oUbNzRTPn5Un+3bYe1a2/fMGfj6dexLVixm5q2o0Lqy0nDRaHrM0aN62XTtYydP6r44e7bz46bBvwz/Dv8yPBy8MA54YRzwwjjghXEgsP4G4n+7Hm3awniMwi8lB7wwDnwD/bFRBvNxDWsAAAAASUVORK5CYII=', 'base64') AS symbol,
+            eigentumsbeschraenkung.aussage_de AS legendetext_de,
+            eigentumsbeschraenkung.artcode,
+            eigentumsbeschraenkung.artcodeliste,
+            eigentumsbeschraenkung.thema,
+            eigentumsbeschraenkung.subthema,
+            eigentumsbeschraenkung.weiteresthema,
+            CASE
+                WHEN geometrie.punkt_lv95 IS NOT NULL
+                    THEN 'Punkt'
+                WHEN geometrie.flaeche_lv95 IS NOT NULL
+                    THEN 'Flaeche'
+             END AS geometrietyp,
+            'ch.SO.'||thema||'.'||weiteresthema AS layername
+        FROM
+            einzelschutz_denkmal_oereb.transferstruktur_eigentumsbeschraenkung AS eigentumsbeschraenkung
+            LEFT JOIN einzelschutz_denkmal_oereb.transferstruktur_geometrie AS geometrie
+            ON eigentumsbeschraenkung.t_id = geometrie.eigentumsbeschraenkung
+    ) AS eigentumsbeschraenkung
+    LEFT JOIN transferstruktur_darstellungsdienst
+    ON transferstruktur_darstellungsdienst.verweiswms ILIKE '%'||RTRIM(eigentumsbeschraenkung.layername||'.'||eigentumsbeschraenkung.geometrietyp, '.')||'%'
+;
+
+UPDATE                                     --???????????????
```

Comment:
Hast du hier eine offene Frage? Ich bin nicht in die Tiefe des SQL eingetaucht und sehe nicht direkt, ob es auch ohne die UPDATE-Statements ginge.